tests/benchdnn/inputs/graph/op/harness_f32_ci

```diff
@@ -58,3 +58,5 @@
 --reset --case=op/f32/select.json
 --reset --case=op/f32/static_reshape.json
 --reset --case=op/f32/static_transpose.json
+--reset --case=op/f32/genindex.json
+--reset --case=op/f32/greaterequal.json
```

Comment:
Can we extend the case to also test broadcasting?

Reply:
thanks for the suggestion, added broadcasting cases for GreaterEqual